src/libmongoc/doc/mongoc_client_encryption_create_encrypted_collection.rst

```diff
@@ -16,7 +16,7 @@ Synopsis
         const bson_t *in_options,
         bson_t *out_options,
         const char *kms_provider,
-        const mongoc_client_encryption_datakey_opts_t *dk_opts,
+        const bson_t *opt_masterKey,
```

Comment:
I suggest adding a way to extend options. When this API becomes stable, it may be helpful to have a way to add supported options. Consider adding a `mongoc_client_encryption_cec_opts_t` type to support only this option:

```rst
:man_page: mongoc_client_encryption_cec_opts_t

mongoc_client_encryption_cec_opts_t
===================================

Synopsis
--------

.. code-block:: c

  typedef struct _mongoc_client_encryption_cec_opts_t mongoc_client_encryption_cec_opts_t;

Used to set options for :symbol:`mongoc_client_encryption_create_encrypted_collection()`.

.. only:: html

  Functions
  ---------

  .. toctree::
    :titlesonly:
    :maxdepth: 1

    mongoc_client_encryption_cec_opts_new
    mongoc_client_encryption_cec_opts_destroy
    mongoc_client_encryption_cec_opts_set_masterkey

.. seealso::

  | :symbol:`mongoc_client_encryption_create_encrypted_collection()`
```

Reply:
Fair point. The option structs may not add much benefit. Adding options to the structs still requires an API addition of the function call. The alternative API addition of a `_v2` / `_v3` may be preferable.